pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "nnunetv2"
```

Comment:
Regarding the `name` parameter in `pyproject.toml` (https://github.com/spinalcordtoolbox/nnUNet-neuropoly/pull/1/commits/932ec6882fcc9ed9869669b1f3d1f348496a896e)

The reason I set the name as `nnunetv2-neuropoly` is so that we can push the package to PyPI under a unique name: https://test.pypi.org/project/nnunetv2-neuropoly/

There is a way to fix the naming conflict. See: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/5003/commits/cd73de2e930fa551ff745ee99a2192ad588bedfa